### PR TITLE
add wildcard match to return metrics

### PR DESCRIPTION
**Pull request by clockfly**
[Review on Reviewable](https://reviewable.io/reviews/intel-hadoop/gearpump/783)

### CODE REVIEW

File: core/src/main/scala/org/apache/gearpump/cluster/ClusterMessage.scala [hunk 1]
```diff
@@ -58,7 +58,7 @@ object ClientToMaster {
 
   case class QueryAppMasterConfig(appId: Int)
 
-  case class QueryHistoryMetrics(appId: Int, path: String)
+  case class QueryHistoryMetrics(appId: Int, path: String, readLatest: Boolean = false)
```
Comment (stanleyxu2005) [inline core/src/main/scala/org/apache/gearpump/cluster/ClusterMessage.scala]: If there is a data type Period or duration, I'd suggest change the method to `case class QueryMetrics(appId: Int, path: String, period: Period = null)`

File: core/src/main/scala/org/apache/gearpump/util/FileServer.scala [hunk 2]
```diff
@@ -79,7 +79,7 @@ class FileServer(rootDir: File, host: String, port : Int) extends Actor with Sta
         case Success(data) =>
           sender ! HttpResponse(entity = HttpEntity(data))
         case Failure(ex) =>
-          LOG.error("get file failed, " + ex.getMessage, ex)
+          LOG.error("get file failed, " + ex.getMessage)
```
Comment (stanleyxu2005) [inline core/src/main/scala/org/apache/gearpump/util/FileServer.scala]: `LOG.error("Failed to get file: " + ex.getMessage)`

**Comment by clockfly** [comment]
It should work now.

**Comment by clockfly** [comment]
It should work now.

**Comment by clockfly** [comment]
It should work now.

**Comment by stanleyxu2005** [comment]
+1

**Pull request merged**